2020/1-year.adoc

```diff
@@ -0,0 +1,44 @@
+= [Current Year] One Year Plan for the People's Liberation Training Committee
+
+:toc:
+:toc-placement!:
+
+toc::[]
```

Comment:
You've got three different TOC things here, I think you've confused the compiler because none of them are showing up, what are you trying to do?